daemon/cmd/daemon_main.go

```diff
@@ -470,9 +470,11 @@ func init() {
 	option.BindEnv(option.K8sServiceCacheSize)
 	flags.MarkHidden(option.K8sServiceCacheSize)
 
+	// Remove in 1.11
 	flags.Bool(option.K8sForceJSONPatch, false, "When set uses JSON Patch to update CNP and CEP status in kube-apiserver")
 	option.BindEnv(option.K8sForceJSONPatch)
 	flags.MarkHidden(option.K8sForceJSONPatch)
+	flags.MarkDeprecated(option.K8sForceJSONPatch, "Marked for removal as it is enabled by default since Kubernetes 1.13")
```

Comment:
Maybe mention that this will be removed in Cilium 1.11 here rather than the comment at line 473?